rendercv/cli/watcher.py

```diff
@@ -0,0 +1,76 @@
+"""
+The `rendercv.cli.watcher` module contains all the functions and classes that are used to watch files and emit callbacks.
+"""
+
+import os
+import pathlib
+import time
+from hashlib import sha256
+from typing import Callable
+
+from watchdog.events import FileModifiedEvent, FileSystemEventHandler
+from watchdog.observers import Observer
+from typer import Exit
+
+
+class ModifiedCVEventHandler(FileSystemEventHandler):
+    """This class handles the file changes and triggers a specified `callback` ignoring duplicate changes.
```

Comment:
Please make sure the maximum width of anything in the source code is below 88 lines. You can use a vertical ruler in VS Code.